src/Components/Artwork/ShelfArtwork.tsx

```diff
@@ -76,7 +77,12 @@ const ShelfArtwork: React.FC<ShelfArtworkProps> = ({
         overflow="hidden"
         {...rest}
       >
-        <Box height={maxImageHeight} display="flex" alignItems="flex-end">
+        <Box
+          height={maxImageHeight}
+          display="flex"
+          alignItems="flex-end"
+          position="relative"
```

Comment:
Also note that in order for this to be positioned correctly, the parent needs to be positioned relatively!